app/views/admin/resource/_index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= link_to "New #{resource_class}", send("new_admin_#{resource_as_sym}_path") %>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Actions</th>
+  </tr>
+  <% collection.each do |resource| %>
+    <tr>
+      <td><%= link_to resource.name, send("admin_#{resource_as_sym}_path", resource) %></td>
```

Comment:
obviously there is any number of attributes that get written here, along with th cells with the names of the attributes. so we need to figure out a way to abstract that.